### DISCUSSION TITLE
Fund not received

### DISCUSSION THREAD

**Discussion by Idris629** [blockscout/blockscout#11363] [General]
I transferred my funds to my trust wallet from the world app but it’s not reflecting in my trust wallet even tho the address is correct. Why is that plz?

**Comment by malcolmxxxx7** [comment]
Sup I transfer money from here to bybyt but not received

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
